app/controllers/qa/linked_data_terms_controller.rb

```diff
@@ -154,7 +154,7 @@ def check_show_subauthority
     end
 
     def create_request_header_service
-      @request_header_service = request_header_service_class.new(request, params)
+      @request_header_service = request_header_service_class.new(request: request, params: params)
```

Comment:
New performance_data_service uses named parameters.  Updating these to be named parameters for consistency.